PopPUNK/__main__.py

```diff
@@ -327,18 +350,32 @@ def main():
         elif args.easy_run:
             sys.stderr.write("Mode: Creating clusters from assemblies (create_db & fit_model)\n")
         if args.r_files is not None:
-            # Sketch
+            # generate sketches and QC sequences
             createDatabaseDir(args.output, kmers)
-            constructDatabase(args.r_files, kmers, sketch_sizes, args.output, args.estimated_length, args.ignore_length, args.threads,
-                args.overwrite)
+            seq_names = constructDatabase(args.r_files, kmers, sketch_sizes,
+                args.output,
+                args.estimated_length,
+                args.threads,
+                args.overwrite,
+                reads = args.reads,
+                strand_preserved = args.strand_preserved,
+                min_count = args.min_kmer_count,
+                use_exact = args.exact_count,
+                qc_filter = args.qc_filter,
+                retain_failures = args.retain_failures,
+                length_sigma = args.length_sigma,
+                lower_length = args.lower_length,
+                upper_length = args.upper_length,
+                prop_n = args.prop_n,
+                upper_n = args.upper_n)
```

Comment:
Better to combine into a single dictionary `qc_dict` perhaps?

Reply:
Use `qc_dict` in 6e49f13.